x11-misc/yabar/yabar-0.4.0_p20190316.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="a0d3fdfed992149b741eb8fcf53f02b5d1a6142e"
+MY_P="yabar-${MY_PV}"
+
+DESCRIPTION="A modern and lightweight status bar for X window managers."
+HOMEPAGE="https://github.com/geommer/yabar"
+SRC_URI="https://github.com/geommer/yabar/archive/${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="playerctl"
+
+DEPEND="
+	x11-libs/cairo[xcb]
+	x11-libs/pango
+	x11-libs/xcb-util-wm
+	x11-libs/gdk-pixbuf
+	x11-libs/libxkbcommon[X]
+	media-libs/alsa-lib
+	net-wireless/wireless-tools
+	dev-libs/libconfig
+	app-text/asciidoc
```

Comment:
Please sort the unconditional deps.
Could you also check all the slot operators and subslot bindings that are missing?

Reply:
Re-ordered and slots considered in newest push